src/guide/testing-the-jdk.md

```diff
@@ -289,7 +289,7 @@ In this example, `MyTest.java` is excluded, tracked by bug `JDK-4711`. `@ignore`
 
 ProblemListing and `@ignore`-ing are done in the JDK source tree, that means a check-in into the repository is needed. This in turn means that a unique JBS issue and a code review are needed. This is a good thing since it makes test problems visible.
 
-* **Code review**: ProblemListing a test is considered a [trivial](#trivial) change.
+* **Code review**: ProblemListing a test is considered a [trivial]#trivial-changes) change.
```

Comment:
```suggestion
* **Code review**: ProblemListing a test is considered a [trivial change]#trivial-changes).
```
I'm still for updating this reference too.

You've marked [my comment](https://github.com/openjdk/guide/pull/156/files#r2258005353) as resolved, yet I can't see the changes.

Reply:
Excuse me, I intended to commit both, but messed up.